docs/README.md

```diff
@@ -1136,6 +1136,7 @@ source <(curl -Ls https://github.com/AdrianDC/android_development_shell_tools/ra
   ```
   * **haste** *&lt;file&gt; or \| haste* [*<b>\[Share&nbsp;hastebin&nbsp;logs\]&nbsp;</b>*](https://github.com/AdrianDC/android_development_shell_tools/blob/master/sources/host/tools.rc)
   * **pbin** *&lt;file&gt; or \| pbin* [*<b>\[Share&nbsp;Pastebin&nbsp;logs\]&nbsp;</b>*](https://github.com/AdrianDC/android_development_shell_tools/blob/master/sources/host/tools.rc)
+  * **dogbin** *&lt;file&gt; or \| dogbin* [*<b>\[Share&nbsp;dogbin&nbsp;logs\]&nbsp;</b>*](https://github.com/AdrianDC/android_development_shell_tools/blob/master/sources/host/tools.rc)
```

Comment:
Use `shtoolsreadme` instead of manually editing the documentation.
https://github.com/AdrianDC/android_development_shell_tools#how-to-contribute-to-the-project